docusaurus.config.js

```diff
@@ -24,7 +24,27 @@ const config = {
   // to replace "en" with "zh-Hans".
   i18n: {
     defaultLocale: 'en',
-    locales: ['en'],
+    locales: ['en', 'es', 'zh-Hans', 'zh-Hant', 'fr', 'de', 'vi', 'th'],
+    localeConfigs: {
+      en: {
+        label: 'English',
+      },
+      es: {
+        label: 'Español',
+      },
+      fr: {
+        label: 'Français',
+      },
+      de: {
+        label: 'Deutsch',
+      },
+      vi: {
+        label: 'Tiếng Việt',
+      },
+      th: {
+        label: 'Thai',
+      },
+    },
```

Comment:
[Question] Why are the list of supported languages are differ from each other?

Reply:
The label is usually given by default by docusaurus if we don't explicitly define them. For Chinese, the default labels were similar to ones in design. Except for Thai, whose default label is **ไทย** instead of Thai, we can remove from localConfigs. Shall I only keep Thai or include all languages for consistency?